pkg/loop/internal/core/services/keyvaluestore/keyvalue_store.go

```diff
@@ -0,0 +1,67 @@
+package keyvaluestore
```

Comment:
@ettec Nit: your package naming is stuttering a bit here. Alternatives could be keyvalue/store.go or similar.